src/slurm_plugin/instance_manager.py

```diff
@@ -903,7 +907,11 @@ def _all_or_nothing_node_assignment(
             # No instances launched at all, e.g. CreateFleet API returns no EC2 instances,
             # or no left instances available from a best-effort EC2 launch
             logger.info("No launched instances found for nodes %s", print_with_count(nodes_resume_list))
-            self._update_failed_nodes(set(nodes_resume_list), "InsufficientInstanceCapacity", override=False)
+            self._update_failed_nodes(
+                set(nodes_resume_list),
+                "InsufficientInstanceCapacity(Check slurm_resume log for ec2 error codes)",
```

Comment:
`InsufficientInstanceCapacity` is. a common pcluster error code used in different places. Can we define a constant for it?

Same for the sentence `Check....codes`. Since we repeat that in many places we can have a constant